src/lib/rules/html-checker/html-checker.ts

```diff
@@ -89,6 +91,11 @@ const rule: IRuleBuilder = {
             };
         };
 
+        const notifyError = async (resource: string, error: any) => {
```

Comment:
Maybe make this function more generic and move it into utils?

Reply:
I think it will complicate things because we will have to pass a lot of variables to have the right context. This is how it could look like:

```ts
await notifyError(debug, context, resource, error, `Couldn't get results for RULENAME for ${resource}`);
```

If we start having to do more things when notifying we could look again into that.

Reply:
ok.